server/routes/changeEstablishment.ts

```diff
@@ -66,6 +66,7 @@ export default function routes({ auditService, supportedPrisonsService, userServ
     const newEstablishment: Prison = {
       prisonId: req.body.establishment,
       prisonName: availablePrisons[req.body.establishment],
+      maxTotalVisitors,
```

Comment:
Setting this `maxTotalVisitors` in the session like we already do for the booking window (`policyNoticeDaysMin`). A limitation of this is that if the configuration changed a user wouldn't get the new values until the logged in again or changed establishment. Unlikely to be an issue in practice as the configuration will very rarely change.

When we implement the DPS header and take out the current establishment switching code we can address this by getting the prison config at the start of a booking journey.